.ci/linux-steps.yaml

```diff
@@ -68,7 +68,9 @@ steps:
   displayName: 'Build'
 
 # Run tests via ctest.
-- script: cd build && CTEST_OUTPUT_ON_FAILURE=1 ctest -T Test .
+# The OMP_NUM_THREADS is to prevent thrashing on older versions of OpenBLAS
+# (0.3.26 and older) where OpenMP and pthreads aren't playing together well.
```

Comment:
Maybe add that a) the default among the available 'alternatives' for `libopenblas-dev` is the `libopenplas-pthread-dev`  and b) that it is this version that exhibits the issue?

Reply:
Also:  Maybe `OMP_NUM_THREADS=2` because `nproc` should yield four at GHA?

Reply:
Sure, I can do that, although do you think I covered that well enough in the documentation changes?

Reply:
I do not think we mention the existence of 'pthread' and 'openmp', their possible co-existence (bad !!) and the fact that fact that just saying 'on ubuntu or debian, get libopenblas-dev' (as the arma docs do) is spelled out clearly enough so an overly casual reader may skip.

The devil really is in the detail, you did really good work narrowing that down, we should say it equally clearly in the docs and I don't think they currently do.  It only needs to be in one place.